extensions/rest/example/grafbase.toml

```diff
@@ -0,0 +1,2 @@
+[extensions]
+rest = { path = "/home/brabier/github/grafbase/grafbase/extensions/rest/build" }
```

Comment:
Should that stay hardcoded to your home path?

Reply:
just a dummy example for us, it's not meant to stay